packages/css/table.css

```
@@ -1,143 +1,110 @@
   --dsc-table-color: var(--ds-color-neutral-text-default);
-  --dsc-table-header-cell-background: var(--ds-color-neutral-background-default);
-  --dsc-table-header-sorted-background: var(--ds-color-neutral-background-subtle);
-  --dsc-table-header-sorted-hover: var(--ds-color-neutral-surface-default);
-  --dsc-table-cell-backround: var(--ds-color-neutral-background-default);
-  --dsc-table-cell-zebra-background: var(--ds-color-neutral-background-subtle);
-  --dsc-table-cell-hover-backround: var(--ds-color-neutral-surface-default);
-
-  position: relative;
-  border-collapse: separate;
-  border-spacing: 0;
-  text-align: left;
-  color: var(--dsc-table-color);
-  width: 100%;
-}
-
-.ds-table--sticky-header {
-  overflow: auto;
-}
-
-.ds-table--border .ds-table__row:last-of-type td {
-  border-bottom: 0;
-}
-
-.ds-table--sm {
-  --dsc-table-padding: var(--ds-spacing-1) var(--ds-spacing-3);
-}
-
-.ds-table--md {
+  --dsc-table-header-background--hover: var(--ds-color-neutral-surface-default);
+  --dsc-table-header-background--sorted: var(--ds-color-neutral-background-subtle);
+  --dsc-table-header-background: var(--ds-color-neutral-background-default);
   --dsc-table-padding: var(--ds-spacing-2) var(--ds-spacing-3);
-}
+  --dsc-table-sort-size: var(--ds-spacing-6);
 
-.ds-table--lg {
-  --dsc-table-padding: var(--ds-spacing-3) var(--ds-spacing-3);
-}
-
-.ds-table__head {
-  z-index: 0;
-  box-sizing: border-box;
-  font-family: inherit;
-  border-spacing: 0;
-  border-bottom: 2px solid var(--dsc-table-border-color);
-}
-
-.ds-table__header__cell {
-  padding: var(--dsc-table-padding);
-  font-family: inherit;
-  background-color: var(--dsc-table-header-cell-background);
+  border-collapse: separate; /* Using separate mode to enable border-radius */
+  border-radius: var(--dsc-table-border-radius);
   border-spacing: 0;
-  border-bottom: 2px solid var(--dsc-table-border-color);
-}
-
-.ds-table--sticky-header .ds-table__head .ds-table__header__cell {
-  position: sticky;
-  top: 0;
-  z-index: 1;
-}
-
-.ds-table__header__cell--sortable {
-  padding: 0;
-}
-
-.ds-table__header__cell--sortable button {
-  position: relative;
-  width: 100%;
-  border: none;
-  font-family: inherit;
-  display: flex;
-  cursor: pointer;
-  gap: var(--ds-spacing-1);
-  align-items: center;
-  padding: var(--dsc-table-padding);
-  background-color: transparent;
   color: var(--dsc-table-color);
-  z-index: 2;
-}
+  width: 100%;
 
-.ds-table__header__cell--sorted button {
-  background-color: var(--dsc-table-header-sorted-background);
-}
+  & > :is(tbody, thead) > tr > :is(th, td) {
+    background: var(--dsc-table-background);
+    border-bottom: 1px solid var(--dsc-table-border-color);
+    padding: var(--dsc-table-padding);
+    text-align: inherit;
+    &:is(th) { font-weight: 500 }
+  }
 
-.ds-table__header__cell--sortable button:focus-visible {
-  z-index: 3;
-  outline-offset: -3px;
-  box-shadow: unset;
-}
+  & > thead > tr > :is(th, td) {
```

```suggestion
  & > thead > :last-child > :is(th, td) {
```

See other comment for context